src/probes/pstore_probe.c

```diff
@@ -230,7 +264,13 @@ int main(int argc, char **argv)
                 // Look for only dmesg logs. Ignore other types of pstore dumps for now.
                 if (sscanf(entry->d_name, "dmesg-efi-%" PRIu64, &id) == 1) {
                         parse_id(id, &count, &part);
-                        telem_debug("DEBUG: Extracted count :%d, part : %d\n", count, part);
+                        telem_debug("dmesg-efi Extracted count :%d, part : %d\n",
+                                    count, part);
+                } else if (sscanf(entry->d_name, "dmesg-ramoops-%" PRIu64, &id) == 1) {
+                        count = id + 1;
```

Comment:
I get this warning:
./../src/probes/pstore_probe.c:270:33: warning: conversion from ‘uint64_t’ {aka ‘long unsigned int’} to ‘int’ may change value [-Wconversion]
  270 |                         count = id + 1;

Reply:
accept